recipes/lyricsmaster/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "lyricsmaster" %}
+{% set version = "2.7.18" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d55fbec619d6096cc789c35070467a5f0ab132464734716590749e75cc3b2bbb" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - lyricsmaster=lyricsmaster.cli:main
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - pip
+    - setuptools
+    - pytest-runner
```

Comment:
you can remove this as well, it's not required for building your package, and neither for running it.

Reply:
Just as a note, anything in `setup_requires` needs to be included in the `host` requirements:

https://github.com/SekouD/lyricsmaster/blob/master/setup.py#L88

Reply:
Thanks @synapticarbors , I didn't understand where this pytest dependency came about in the recipe.

Now I know :)